### PR TITLE
Fix mif-file generation

### LINKED ISSUE #108: Mif-file generation broken
The changes in `micromagneticmodel.term.__contains__` (checking for all attributes) introduced in ubermag/micromagneticmodel#46 breaks the `mif` file generation. In `driver.py:driver_script` we use `mm.Damping() in system.dynamics` tests (similar for `mm.Precession`). Without the correct arguments these tests fail and we write wrong `alpha` values to the `mif` file.

### PR DESCRIPTION
**Pull request by lang-m**
Closes #108

### CODE REVIEW

File: oommfc/scripts/driver.py [hunk 1]
```diff
@@ -109,9 +109,9 @@ def driver_script(
                     "be specified explicitely."
                 )
                 raise RuntimeError(msg)
-            elif mm.ZhangLi() in system.dynamics:
+            elif system.dynamics.get(type=mm.ZhangLi):
```
Comment (marijanbeg) [inline oommfc/scripts/driver.py:112]: If you decide to go for `Container` as the return type of `get`, then `Container` would need `__bool__` to allow this logic.
```python
def __bool__(self):
    return bool(self._terms)
```

File: oommfc/scripts/driver.py [hunk 2]
```diff
@@ -131,43 +131,44 @@ def driver_script(
             raise TypeError(msg)
 
         # Extract dynamics equation parameters.
-        if mm.Precession() in system.dynamics:
+        if system.dynamics.get(type=mm.Precession):
+            precession = system.dynamics.get(type=mm.Precession)[0]
```
Comment (marijanbeg) [inline oommfc/scripts/driver.py:135]: Slightly more obvious with unpacking maybe?

`[precession] = system.dynamics.get(type=mm.Precession)`

or 

`(precession,) = system.dynamics.get(type=mm.Precession)`

File: oommfc/scripts/driver.py (hunk 1 above)
Comment (marijanbeg) [reply oommfc/scripts/driver.py:112]: Aha, I did not know we have `__len__` for container :)

**Review by swapneelap — APPROVED** [review]
Looks good to me.

**Pull request merged**